src/main/scala/chisel3/testing/scalatest/HasConfigMap.scala

```diff
@@ -0,0 +1,48 @@
+// SPDX-License-Identifier: Apache-2.0
+
+package chisel3.testing.scalatest
+
+import org.scalatest.{ConfigMap, TestSuite, TestSuiteMixin}
+import scala.util.DynamicVariable
+
+/** A Scalatest test suite mix-in that provides access to command line options.
+  *
+  * This expose any keys/values passed using the `-D<key>=<value` command line
+  * option to the test via a `Map`.
+  *
+  * For example, you can invoke Scalatest passing the `foo=bar` option like so:
+  * {{{
+  * ./mill 'chisel[2.13.16].test.testOnly' 'fooTest' -Dfoo=bar
```

Comment:
```suggestion
  * ./mill chisel[2.13.16].test.testOnly fooTest -Dfoo=bar
```

This is the nit of nits but those quotes aren't necessary

Reply:
SGTM. Those quotes are unnecessary. Fixed in: 70c114bc9